.github/workflows/test-and-publish.yml

```
@@ -18,22 +20,38 @@ jobs:
         run: docker compose -f actions-services.yml run --rm app ./check-psr2.sh
 
   build-and-publish:
-    name: Build and Publish
+    name: Build and Publish Docker image 
     needs: tests
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
+
       - name: Log in to Docker Hub
         uses: docker/login-action@v3
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
+      
+      - name: Log in to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+            registry: ghcr.io
+            username: ${{ github.actor }}
+            password: ${{ secrets.GITHUB_TOKEN }}
+
       - name: Extract metadata (tags, labels) for Docker
         id: meta
         uses: docker/metadata-action@v5
         with:
-          images: ${{ vars.DOCKER_ORG }}/${{ github.event.repository.name }}
+          images: |
+           ${{ vars.DOCKER_ORG }}/${{ github.event.repository.name }}
+           ghcr.io/${{ github.repository }}
```

In your editor, there may be a plugin that can format yml in a standard way. Ask me later if you're interested.
```suggestion
            ${{ vars.DOCKER_ORG }}/${{ github.event.repository.name }}
            ghcr.io/${{ github.repository }}
```